src/components/10-atoms/input-text/index.js

```diff
@@ -104,15 +173,30 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       // yes, set UI from model state
       this.nativeInput.value = this.modelValue;
     }
+
+    if (this.maxLength) {
+      this.modelCounter = this.getCounterText;
+    }
   };
 
+  isNotAtMaxLength() {
+    return this.charsLeft !== 0 && this.charsLeft !== '0';
+  }
+
   firstUpdated() {
-    const { defaultValue, isReact } = this;
+    const { defaultValue, isReact, value } = this;
     this.nativeInput = this.querySelector('input');
 
+    if (isReact) {
```

Comment:
This smells DRY violation. I suggest `if (isReact) { this.nativeInput.value = defaultValue || value; }`

Reply:
This also is not my code, so I better do not touch it within this PR. Not that I introduce potential other bugs.